src/component/toolbox/ToolboxModel.ts

```diff
@@ -31,6 +31,7 @@ import {
     CommonTooltipOption,
     Dictionary
 } from '../../util/types';
+import GlobalModel from "../../model/Global";
```

Comment:
Note that we should use `'` but not `"`, check everywhere and correct them, please.

Reply:
@plainheart Thanks for the detailed review. I just fixed it